### ISSUE TITLE
`npm run serve` error with latest dependencies

### ISSUE THREAD

**Issue by peterdesmet** [inbo/crow#185]
When running `npm update` to get the latest dependencies and then trying `npm run serve`, we get the following error:

```json
 ERROR  Error: Rule can only have one resource source (provided resource and test + include + exclude) in {
  "exclude": [
    null
  ],
  "use": [
    {
      "loader": "/Users/peter_desmet/Coding/Repositories/inbo/crow/node_modules/@vue/cli-plugin-babel/node_modules/cache-loader/dist/cjs.js",
      "options": {
        "cacheDirectory": "/Users/peter_desmet/Coding/Repositories/inbo/crow/node_modules/.cache/babel-loader",
        "cacheIdentifier": "51466cbc"
      },
      "ident": "clonedRuleSet-39.use[0]"
    },
    {
      "loader": "/Users/peter_desmet/Coding/Repositories/inbo/crow/node_modules/babel-loader/lib/index.js",
      "options": "undefined",
      "ident": "undefined"
    }
  ]
}
```

This is only an issue _if_ we want to update to the latest dependencies (i.e. by running a fresh install without a `package-lock.json`)

**Comment by peterdesmet** [comment]
@niconoe any insight you could provide?

**Comment by niconoe** [comment]
Starting with a fresh install, can confirm I have the same error than you (after fol,lowing your steps from #184). Now investigating for a solution!

**Comment by peterdesmet** [comment]
Fixed in #187
